README-kr

```diff
@@ -0,0 +1,1784 @@
+# What the f\*ck JavaScript?
+
+[![WTFPL 2.0][license-image]][license-url]
+[![NPM version][npm-image]][npm-url]
+
+> 재미있고 교묘한 JavaScript 예제
+
+JavaScript는 훌륭한 언어입니다. JavaScript는 구문이 단순하며 큰 생태계를 가지고 있습니다. 가장 중요한 점은 훌륭한 공동체를 가지고 있다는 것입니다.
+
+동시에, 우리 모두는 자바스크립트가 까다로운 부분을 가진 꽤 재미있는 언어라는 것을 알고 있습니다. 몇몇 특징은 우리의 일상적인 일을 순식간에 지옥으로 바꾸기도 하고, 우리를 크게 웃게 만들기도 합니다.  
+
+WTFJS의 아이디어는 [Brian Leroux](https://twitter.com/brianleroux)에 속해있습니다. 이 목록들은 그의 이야기에서 꽤 영감을 받았습니다. [**“WTFJS”** at dotJS 2012](https://www.youtube.com/watch?v=et8xNAc2ic8):
+
+[![dotJS 2012 - Brian Leroux - WTFJS](https://img.youtube.com/vi/et8xNAc2ic8/0.jpg)](https://www.youtube.com/watch?v=et8xNAc2ic8)
+
+# Node Packaged Manuscript
+
+이 핸드북은 `npm`를 이용하여 설치할 수 있습니다. 그냥 실행합시다:
+
+```
+$ npm install -g wtfjs
+```
+
+이제 당신은 커맨드 창에서 'wtfjs'를 실행할 수 있게 되었습니다. 당신이 선택한 '$PAGER'에서 열릴 것 입니다. 그렇지 않으면 계속 여기서 읽어도 됩니다. 
+
+출처는 <https://github.com/denysdovhan/wtfjs> 여기에서 확인 할 수 있습니다. 
+
+# 번역
+
+현재, **wtfjs**는 아래와 같은 언어로 번역되었습니다.:
+
+- [中文版](./README-zh-cn.md)
+- [Français](./README-fr-fr.md)
+- [Português do Brasil](./README-pt-br.md)
+- [Polski](./README-pl-pl.md)
+- [Italiano](./README-it-it.md)
+
+[**다른 번역 **][tr-request]
+
+[tr-request]: https://github.com/denysdovhan/wtfjs/issues/new?title=Translation%20Request:%20%5BPlease%20enter%20language%20here%5D&body=I%20am%20able%20to%20translate%20this%20language%20%5Byes/no%5D
+
+<!-- prettier-ignore-start -->
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+# 목차
+
+- [💪🏻 Motivation](#-motivation)
+- [✍🏻 Notation](#-notation)
+- [👀 Examples](#-examples)
+  - [`[]`와 `![]`은 같다](#-is-equal-)
+  - [`true`는 `![]`와 같지 않지만, `[]` 이와도 같지 않다](#true-is-not-equal--but-not-equal--too)
+  - [true는 false이다](#true-is-false)
+  - [baNaNa](#banana)
+  - [`NaN`은 `NaN`이 아니다](#nan-is-not-a-nan)
+  - [이것은 실패다](#its-a-fail)
```

Comment:
omitted [Object.is() and === weird cases](https://github.com/denysdovhan/wtfjs#objectis-and--weird-cases)

better be
> [Object.is()와 ===에 있어서 이상한 경우들]